cms/envs/common.py

```diff
@@ -615,10 +615,14 @@
 ############################ DJANGO_BUILTINS ################################
 # Change DEBUG in your environment settings files, not here
 DEBUG = False
+
+# See comments in the auth_provider configuration.
 SESSION_COOKIE_SECURE = False
 SESSION_SAVE_EVERY_REQUEST = False
 SESSION_SERIALIZER = 'django.contrib.sessions.serializers.PickleSerializer'
 
+# In production, redefined by aws.py
+SESSION_COOKIE_NAME = 'signed_sessionid'
```

Comment:
Thank you @imankulov, this change is breaking many tests since they use the default name `sessionid`. And as you stated; the deployment configuration repo is overriding it already

This is an example of a hard-coded use of the original name in tests
https://github.com/Edraak/edraak-platform/blob/dd0180a33d0662cb4105cee7a5f97817d69db3a9/common/djangoapps/student/tests/test_auto_auth.py#L259


You can do one of the following (personally arranged by most favorite options, to least favorite):
* Skip this change and close the PR
* Add a fix line in test env so tests can pass https://github.com/Edraak/edraak-platform/blob/master/cms/envs/test.py
```python
SESSION_COOKIE_NAME = 'sessionid'
```
* Fix 18 failing tests (@qutaiba can give you the list of failing tests, or access to Jenkins)

Reply:
Oh, indeed, I missed the failing tests thing, and I don't have access to Jenkins. Thanks for pointing this out, @shadinaif.

If we close the PR without the changes, we end up with broken development installations, which is sub-optimal. The least intrusive solution would probably be to update "progs" and "mktg" to consider both `signed_sessionid` and `sessionid` cookie names as valid.